rotkehlchen/utils/misc.py

```diff
@@ -384,3 +384,28 @@ def create_order_by_rules_list(
     order_by_attribute = data['order_by_attributes'] if data['order_by_attributes'] is not None else [timestamp_field]  # noqa: E501
     ascending = data['ascending'] if data['ascending'] is not None else [False]
     return list(zip(order_by_attribute, ascending))
+
+
+def levenshtein_distance(s1: str, s2: str) -> float:
+    """Calculate the Levenshtein distance between two strings.
+    Code is taken from:
+    https://en.wikibooks.org/wiki/Algorithm_Implementation/Strings/Levenshtein_distance#Python
+    """
```

Comment:
Can you add a little test for this function to verify that works as expected?

Reply:
sure